device/arista/x86_64-arista_7170_64c/Arista-7170-Q59S20/buffers_defaults_t0.j2

```diff
@@ -50,7 +51,7 @@
         "ingress_lossless_profile": {
```

Comment:
No need to retain the ingress_lossless_profile as buffermgrd will auto generate the profile using pg profile lookup file based on the line rate and the cable length.